gpflow/kernels/conditioned.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+import tensorflow as tf
+from .base import Kernel
+from ..config import default_float, default_jitter
+
+
+class Conditioned(Kernel):
```

Comment:
One of the criteria for merging PRs is test cases for new features. Could you add tests?

Reply:
added one!